docs/sources/flow/tasks/configure/configure-kubernetes.md

```diff
@@ -21,6 +21,12 @@ weight: 200
 
 To configure {{< param "PRODUCT_NAME" >}} on Kubernetes, perform the following steps:
 
+TODO(thampiotr): The values.yaml is quite dense and may not be suitable for new users. Instead, we should
```

Comment:
Is this description more of a tutorial than task? It also might be to broad, when saying configure kubernetes, is that for metrics, logs, traces ,pyroscope , mimir rules? Each of the them should be a different task since they require substantially different approaches.

Reply:
Basically I think the below `collect-` are good and this one is a tutorial

Reply:
Yeah, things are getting more complicated when I realise that to deploy as a useful DaemonSet for logs, one needs to install the helm chart together with the correct volume mounts etc. 

Having it all on a single `collect-` page may be too much though, I was wondering if I can split it into two pages: one to install a DaemonSet with Helm, and second one to start using the kubernetes module from agent-modules repo for logs. 

I'll maybe start with a single large page and we'll split this later if needed. Will be easier to decide once we have a full doc.

Reply:
Thanks @mattdurham ! I've updated this now to reflect this slightly different, but I think better approach.